lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -1680,6 +1726,9 @@ namespace Js
 
         ScriptContext* scriptContext = arrayPrototype->GetScriptContext();
         JavascriptLibrary* library = arrayPrototype->GetLibrary();
+
+        library->EnsureIndexOfByteCode();
```

Comment:
This is ok now, but we may want to make more generic when we start adding more builtins.